src/main/kotlin/com/terraformation/backend/seedbank/api/ViabilityTestsController.kt

```diff
@@ -133,7 +135,7 @@ data class GetViabilityTestPayload(
       seedsCompromised = model.seedsCompromised,
       seedsEmpty = model.seedsEmpty,
       seedsFilled = model.seedsFilled,
-      seedsTested = model.seedsTested,
+      seedsTested = model.seedsTested ?: 1,
```

Comment:
is this to handle existing null data?

Reply:
Yes. There's only one case in our existing data across prod and staging where this is null, and it's a test accession, so it didn't seem worth doing something more sophisticated here.

Reply:
I should add that when we get rid of the v1-only parts of the data model, this will become non-nullable in the database and we'll clean up the existing data at that point. I considered doing that here but I don't want to break v1's contract which allows this to be null. Hopefully nobody else will actually create a viability test without a seed count between now and then.